### PR TITLE
Do not emit label with select when not given

### PR DESCRIPTION
**Pull request by berenddeboer**
A minor improvement.

### CODE REVIEW

**Comment by aforemny** [comment]
Hi @berenddeboer, is there any indication in Mdc web that a select without a label is a valid pattern? I searched a little and could not find anything.

**Comment by berenddeboer** [comment]
It might not be very common, but I had some layouts where clearly a label did not make sense. I.e. the label was a pretty long description, with a select below where a label didn't add anything. Not superimportant, I created this as I was struggling with an issue on iOS/Safari, and thought removing the label might help (it didn't).

An empty label clearly is possible, and it also doesn't add anything, so that's why I thought why even include an empty label.

**Comment by berenddeboer** [comment]
Hmm, not sure I made this branch correctly. Let me know if you're happy with the changes, and I'll create a new branch based on master, I think this may have some old slider stuff in it. I tried to rebase but not sure that worked well.

File: README.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ view : Model -> Html Msg
 view model =
     Html.div []
         [
-          Button.view Mdc [0] model.mdc
+          Button.view Mdc "my-button"" model.mdc
```
Comment (SidneyNemzer) [inline README.md]: Is that an extra quote? :)

**Comment by aforemny** [comment]
Hi @berenddeboer, I will close here because the issue is stale. Sorry I did not get to this. If you would like me to reconsider, maybe hit me up on Slack so that we can discuss what this change actually means? :-)

**Pull request closed without merging**